__tests__/app.test.js

```diff
@@ -62,3 +63,43 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id", () => {
+  test("GET: 200 - SHOULD RESPOND WITH AN OBJECT CONTAINING SEVERAL REVIEW PROPERTIES", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
Are we expecting any number for the id in this test? Be sure that it's checking specifically what you need on the object for parametric endpoints